docs/design/proposals/encryption-with-vault-tokens.md

```diff
@@ -180,8 +180,73 @@ data:
   vaultBackendPath: "secret/ceph-csi-encryption/"
   vaultTLSServerName: "vault.infosec.example.org"
   vaultCAFromSecret: "vault-infosec-ca"
+  vaultClientCertFromSecret: "vault-client-cert"
+  vaultClientCertKeyFromSecret: "vault-client-cert-key"
   vaultCAVerify: "true"
 ```
 
 Only parameters with the `vault`-prefix may be changed in the Kubernetes
 ConfigMap of the Tenant.
+
+### Certificates stored in the Tenants Kubernetes Namespace
+
+The `vaultCAFromSecret` , `vaultClientCertFromSecret` and
+`vaultClientCertKeyFromSecret` secrets should be created in the namespace where
+cephcsi is deployed. The sample of secrets for the CA and client Certificate.
+
+#### CA Certificate to verify Vault server SSL certificate
+
+```yaml
+---
+apiVersion: v1
+kind: secret
+metadata:
+  name: vault-infosec-ca
+data:
```

Comment:
Because this is `data`, the contents needs to be base64 encoded. Maybe use `stringData` so that the contents can be placed plain-text? Either way, it probably needs mentioning.

Reply:
changing it to stringData, as its standard in kubernetes to mention the plain-text I think we don't need to mention it explicitly.